src/EditorFeatures/Core/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -97,13 +116,25 @@ private bool CompleteComment(
             if (snippet != null)
             {
                 ApplySnippet(snippet, subjectBuffer, textView);
+                var oldSnapshot = subjectBuffer.CurrentSnapshot;
+                var oldCaret = textView.Caret.Position.VirtualBufferPosition;
+
                 returnValue = true;
+
+                var token = _asyncListener.BeginAsyncOperation(nameof(GenerateDocumentationCommentProposalsAsync));
```

Comment:
i think we can move all of this into _generateDocumentationCommentManager, say in a method `TriggerDocumentationProposalGeneration`